src/config/AppConfiguration.vanilla.js

```diff
@@ -0,0 +1,52 @@
+// Copyright (c) Cosmo Tech.
+// Licensed under the MIT license.
+
+// To configure your app see:
+// https://github.com/Cosmo-Tech/azure-sample-webapp/blob/main/doc/appConfiguration.md
+
+// App version
+export const APP_VERSION = process.env.REACT_APP_VERSION;
+
+// App translation configuration
+export const LANGUAGES = {};
+
+export const FALLBACK_LANGUAGE = ''; // Language used if detection fails
```

Comment:
We could keep the values defined for both languages currently supported by the sample webapp:
```
// App translation configuration
export const LANGUAGES = {
  en: 'English',
  fr: 'Français',
};
export const FALLBACK_LANGUAGE = 'en'; // Language used if detection fails
```

The goal of the "vanilla" version is to remove the elements that are specific to the brewery solution, but we should keep the parts that are "common" to all solutions and can be used as default values when creating a new webapp (e.g. the "en" and "fr" languages that are already supported).